playwright-codecept-js/github_test.js

```diff
@@ -0,0 +1,10 @@
+Feature('GitHub Login');
+
+Scenario('login on GitHub', ({ I }) => {
```

Comment:
Can you add multiple scenarios and ar least 2 feature files? Also add config to run those in parallel.

Reply:
@ankur-lt need to try this. This might take some time. As user requested for sample, so created the simple one. Regarding the parallel & multiple configs for one test we can take it later.